quetz/jobs/models.py

```diff
@@ -62,6 +65,8 @@ class Job(Base):
         default=JobStatus.pending,
     )
 
+    tasks = sa.orm.relationship('Task', cascade="all,delete-orphan")
```

Comment:
@btel is this fine? I changed the relationship / backref stuff to appease mypy.

Reply:
yes, it should be the same ;-)